internal/expr/query.go

```diff
@@ -151,6 +151,7 @@ func (qe *QueryExpr) ScanArgs(columns []string, outputArgs []any) (scanArgs []an
 
 	// Generate the pointers.
 	var ptrs = []any{}
+	var columnInResult = make(map[int]bool)
```

Comment:
Sorry for missing this earlier, but this can be just a simple slice:
```
var columnInResult = make([]bool, len(qe.outputs))
```

Reply:
Good point, changed from map to slice.